R/Spectra.R

```diff
@@ -424,6 +424,8 @@ NULL
 #'
 #' @exportMethod Spectra
 #'
+#' @import MsCoreUtils
```

Comment:
I'm not so happy with importing the whole `MsCoreUtils` namespace - I would rather selectively import only functions needed, and have that import in the header of the function that is using it.

Reply:
Yes, selective importing relevant functions is better. I needed a quick fox to get the webpage to build.